libs/gltfio/src/ResourceLoader.cpp

```diff
@@ -485,7 +491,9 @@ void ResourceLoader::computeTangents(FFilamentAsset* asset) const {
         helper.getQuats(quats, vertexCount);
 
         // Upload quaternions to the GPU.
-        auto callback = (VertexBuffer::BufferDescriptor::Callback) free;
+        VertexBuffer::BufferDescriptor::Callback callback = [](void* mem, size_t, void*) {
```

Comment:
Could we define a single callback function and reuse it each time in this file?

Reply:
Good idea, done.